phpBB/assets/javascript/plupload.js

```diff
@@ -446,6 +446,44 @@ phpbb.plupload.fileError = function(file, error) {
 phpbb.plupload.uploader = new plupload.Uploader(phpbb.plupload.config);
 phpbb.plupload.initialize();
 
+/**
+ * Add a file filter to check for max file sizes per mime type.
+ */
+plupload.addFileFilter('mime_types_max_file_size', function(types, file, cb) {
+	if (file.size !== 'undefined') {
+		$(types).each(function(i, type) {
+			let extensions = [],
+				exts_array = type.extensions.split(',');
```

Comment:
We're using camelCase in JS
```suggestion
				extsArray = type.extensions.split(',');
```